azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -146,6 +148,14 @@ protected FunctionRuntimeHandler getFunctionRuntimeHandler() throws AzureAuthFai
             case Linux:
                 builder = new LinuxFunctionRuntimeHandler.Builder();
                 break;
+            case Docker:
+                final RuntimeConfiguration runtime = this.getRuntime();
+                builder = new DockerFunctionRuntimeHandler.Builder()
+                        .image(runtime.getImage())
+                        .serverId(runtime.getServerId())
+                        .registryUrl(runtime.getRegistryUrl())
+                        .functionExtensionVersion(getFunctionExtensionVersion());
```

Comment:
should report when getFunctionExtensionVersion returns null or empty

Reply:
Will create a enum for FunctionExtensionVersion and do the validation